routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: certified-copies.orders.api.ch.gov.uk
+group: api
+weight: 924
+routes:
+   1: ^/orderable/certified-copies/.*
```

Comment:
Not entirely necessary, but it says there is no new line at the end of the file

Reply:
A new line is forthcoming then :)